src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,21 +143,26 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
-    let dropdownItems = [
-      {
-        className: 'hidden',
-        html: 'More',
-        id: JobActionItem.MORE
-      },
-      {
+    let job = MetronomeStore.getJob(this.props.params.id);
+    let dropdownItems = [];
```

Comment:
what do you think about just having the hidden/destroy objects in the array right when it's invoked so we don't have to do pushing, then inside of the check do a `splice` with the correct index?